### ISSUE TITLE
Allow import of RAT result also in the firefox browser

### ISSUE THREAD

**Issue by DerVogel2020** [corona-warn-app/cwa-wishlist#532]
## Current Implementation
Currently, on android smartphones, only chrome is supported, on iOS, only safari is supported, see https://github.com/corona-warn-app/cwa-event-landingpage/blob/d05ef2af9ca61457b7c7affe01b4ec94f2d7700d/src/data/index.json and https://github.com/corona-warn-app/cwa-event-landingpage/blob/d05ef2af9ca61457b7c7affe01b4ec94f2d7700d/src/data/index_de.json

All restrictions can be seen in app.js here: https://github.com/corona-warn-app/cwa-event-landingpage/blob/d05ef2af9ca61457b7c7affe01b4ec94f2d7700d/src/assets/js/app.js

## Suggested Enhancement
Allow also other browser, IMHO at least firefox, to import the RAT result into the CWA.

## Expected Benefits
More users can directly import the RAT result into the CWA, as not all android users do have chrome installed.

----
Internal Tracking-ID: [EXPOSUREAPP-7981](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-7981)

**Comment by Ein-Tim** [comment]
I think under iOS we have the problem that the app links only works correctly in Safari. 

Related issues:

- https://github.com/corona-warn-app/cwa-event-landingpage/issues/22
- https://github.com/corona-warn-app/cwa-event-landingpage/issues/23

**Comment by DerVogel2020** [comment]
Thanks. It's quite difficult to search all repositorys to find a related issue.

**Comment by DerVogel2020** [comment]
Update: It's possible in firefox to hold the link at the test result page from the testing site. Then the link "https://s.coronawarn.app/?v=1#..." is shown. Clicking on "Link in externer App öffnen" opens the CWA and the possibility to import the test result into the CWA.
I think, a faq entry on the website would be enough.

**Comment by thomasaugsten** [comment]
Ok we will add this information also to the landing page

**Comment by dsarkar** [comment]
@DerVogel2020 Thanks for your last comment. Questions: 

1. This provider does not sent you the link via email? 
2. If yes, if you click the link in the email, does this work (CWA opens and offers to add test)?
3. So, if I understand correctly, this provider asked you to "log" into their page and then you get a link to add the test?
4. If the latter is correct, they should also mention on their page the "long-press" workaround (not only FAQ), right?
5. Which provider are we talking about?

Thanks. Best, DS

---
Corona-Warn-App Open Source Team

**Comment by DerVogel2020** [comment]
@dsarkar 
I got an email with a link to the website of the provider https://schnelltest.io/labs/testzentrum-drive-stuhr/tests (answering 1. and 5.)
3. Yes, I have to enter a code (I got this at the testing site) and my e-mail-address used at the booking of the test.
4. Yes, that would be a solution, only on smartphones, because I cannot scan a qr code shown at the display with the camera of the same device.

Just discovered: I just clicked the link in the e-mail under Windows and entered my code and e-mail-address and clicked the button "Ergebnis in App eintragen" in firefox, a qr code is shown with the hint "Offne deine Corona Warn App um den QR Code zu scannen".
